docs/src/man/basics.md

```diff
@@ -0,0 +1,1220 @@
+
+The indexing syntax can also be used to select rows based on conditions on variables:
+```jldoctest dataframe
+julia> german[german.id .> 600, :]
+399×10 DataFrame
+ Row │ id     Age    Sex     Job    Housing  Saving accounts  Checking account ⋯
+     │ Int64  Int64  String  Int64  String   String           String           ⋯
+─────┼──────────────────────────────────────────────────────────────────────────
+   1 │   601     30  female      2  own      little           moderate         ⋯
+   2 │   602     34  female      1  free     little           moderate
+   3 │   603     28  female      3  own      little           NA
+   4 │   604     23  female      2  own      little           rich
+   5 │   605     22  male        2  own      quite rich       little           ⋯
+   6 │   606     74  male        3  own      little           NA
+   7 │   607     50  female      2  free     moderate         moderate
+   8 │   608     33  male        2  own      little           NA
+  ⋮  │   ⋮      ⋮      ⋮       ⋮       ⋮            ⋮                ⋮         ⋱
+ 393 │   993     30  male        3  own      little           little           ⋯
+ 394 │   994     50  male        2  own      NA               NA
+ 395 │   995     31  female      1  own      little           NA
+ 396 │   996     40  male        3  own      little           little
+ 397 │   997     38  male        2  own      little           NA               ⋯
+ 398 │   998     23  male        2  free     little           little
+ 399 │   999     27  male        2  own      moderate         moderate
+                                                  3 columns and 384 rows omitted
+```
+
+If you need to match a specific subset of values, then `in()` can be applied:
+```jldoctest dataframe
+julia> german[in.(german.id, Ref([1, 6, 908, 955])), :]
+5×10 DataFrame
+ Row │ id     Age    Sex     Job    Housing  Saving accounts  Checking account ⋯
+     │ Int64  Int64  String  Int64  String   String           String           ⋯
+─────┼──────────────────────────────────────────────────────────────────────────
+   1 │     1      1  male        2  own      NA               little           ⋯
+   2 │     1      1  female      2  own      little           moderate
+   3 │     6     53  male        2  own      quite rich       NA
+   4 │   908     46  female      1  own      little           NA
+   5 │   955     57  female      3  rent     rich             little           ⋯
+                                                               3 columns omitted
+```
+
+Equivalently, the `in` function can be called with a single argument to create a function object that
+tests whether each value belongs to the subset (partial application of `in`):
+`german[in([1, 6, 908, 955]).(german.id), :]`
+
+## Views
+
+You can simply create a `view` of a DataFrame (it is more efficient than creating a materialized selection).
+Here are the possible return value options.
+
+```jldoctest dataframe
+julia> @view german[1:5, 1]
```

Comment:
maybe start with more standard examples. You are showing corner cases first now.

Reply:
Standard examples like?

Reply:
this is not a view of a data frame, but a view of its column. A more standard thing to show first would be a `SubDataFrame`